www/__mocks__/cordovaMocks.ts

```diff
@@ -117,6 +129,16 @@ export const mockBEMUserCache = () => {
         return false;
       }
     },
+    getAllTimeQuery: () => {
+      return { key: 'write_ts', startTs: 0, endTs: Date.now() / 1000 };
+    },
+    getSensorDataForInterval: (key, tq, withMetadata) => {
+      return new Promise<any>((rs, rj) =>
+        setTimeout(() => {
+          rs({ metadata: { write_ts: '1699897723' }, data: 'completed', time: '01/01/2001' });
+        }, 100),
```

Comment:
We should only resolve with this if the key is `manual/demographic_survey`

Reply:
done in [8b3ab76](https://github.com/e-mission/e-mission-phone/pull/1063/commits/8b3ab76327d93011e2a31cfa6409bcb23f710e1f)